src/lib/y2partitioner/widgets/pages/page.rb

```diff
@@ -0,0 +1,48 @@
+# Copyright (c) [2020] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "cwm/widget"
```

Comment:
Are all these requires needed?

Reply:
I think so. They were there in almost all pages inheriting from CWM::Page.

Anyway, let's re-check it.

Reply:
We need at least to require [`cwm/tree_pager`](https://github.com/yast/yast-yast2/blob/master/library/cwm/src/lib/cwm/tree_pager.rb), which requires `cwm/page`, `cwm/pager`, and `cwm/tree`. Or require them explicitly.

Reply:
I mean, to require both, `cwm/page` and `cwm/tree_pager`